cmd/reviewdog/main.go

```diff
@@ -316,18 +316,26 @@ func run(r io.Reader, w io.Writer, opt *option) error {
 			return runDoghouse(ctx, r, w, opt, isProject)
 		}
 		var err error
-		checkService, ghDiffService, err := githubCheckService(ctx, opt)
+		var isPR bool
+		checkService, ghDiffService, isPR, err := githubCheckService(ctx, opt)
 		if err != nil {
 			return err
 		}
+		if !isPR {
+			opt.filterMode = filter.ModeNoFilter
+		}
 		ds = ghDiffService
 		cs = reviewdog.MultiCommentService(checkService, cs)
 	case "github-pr-annotations":
```

Comment:
Can you introduce "github-annotations" reporter, similar to github-check to github-pr-check?

Reply:
Sure, done. While doing so, I realized that `github-check` and `github-pr-check` are currently identical. I added a check that will fail if `github-pr-[check|annotations]` is run on a non-PR. Since this might be a breaking change, let me know if I should revert it.